docs/pages/management/admin/adding-nodes.mdx

```diff
@@ -134,7 +134,7 @@ Execute the following commands on the host where you will run your Node so you
 can use the `CA_PIN` and `INVITE_TOKEN` variables to start Teleport.
 
 ```code
-$ export CA_PIN=$<CA pin>
+$ export CA_PIN=<CA pin>
 $ export INVITE_TOKEN=<invite token>
```

Comment:
Should we use `Var` components here, e.g., `export CA_PIN=<Var name="ca-pin" />`?

Reply:
dd401ec1948744b9d5d90f0091820322e40d7869